.github/workflows/ci.yml

```diff
@@ -183,12 +183,21 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - name: Test on node.js ${{ matrix.node-version }}
+        if: matrix.node-version == '8'
```

Comment:
Let's also skip them on Node.js 6 and 10: they are painfully slow, and we don't get much benefits from also running fuzz tests there :sweat_smile: 

If/when we'll add more combinations, if it becomes too slow then we can also decide to only run fuzz tests on a single Node.js version.

Reply:
Is there a way to set `TEST_FUZZ` in an earlier rule so that it's true for 12/14/16 and false for 6/8/10?

Reply:
Maybe this, but I didn't try:
```yaml
      - name: Test on node.js ${{ matrix.node-version }}
        # Hack: --color has supports-color@5 returned true for GitHub CI
        # Remove once `chalk` is bumped to 4.0.

        # Todo(Babel 8): Jest execution path is hardcoded because Yarn 2 does not support node 6
        run: |
          node --max-old-space-size=4096 ./node_modules/.bin/jest --ci --color
        with:
          BABEL_ENV: test
          TEST_FUZZ: ${{ matrix.node-version >= 12 }}
```

Reply:
Done: https://github.com/babel/babel/pull/14201/commits/1a24870b4b7b968cbb71239eeeb7cd19ff401fde